README.md

```diff
@@ -26,72 +26,71 @@ The sync engine itself is built on top of a few third-party frameworks, and uses
 
 ![Mobile app architecture](https://github.com/wireapp/wire/blob/master/assets/mobile-architecture.png?raw=true)
 
+### Documentation
```

Comment:
Do we really have to revert all this?
Can't you just replace 15.4 by 15.3?

Reply:
I just used the GitHub feature "Revert" button... but yeah I can change it